install_config/certificate_customization.adoc

```diff
@@ -226,17 +226,17 @@ Where the parameter values are:
 
 * *cafile* is the path to the file that contains the root CA for this key and certificate. If an intermediate CA is in use, the file should contain both the intermediate and root CA.
 
-If these certificate files are new to your {product-title} cluster, run the Ansible *_deploy_cluster.yml_* playbook to add these files to the {product-title} configuration files. 
-The playbook adds the certificate files to the *_/etc/origin/master/named_certificates/_* directory.
+If these certificate files are new to your {product-title} cluster, run the Ansible *_deploy_router.yml_* playbook to add these files to the {product-title} configuration files. 
+The playbook adds the certificate files to the *_/etc/origin/master/_* directory.
 
 ----
 ifdef::openshift-enterprise[]
 # ansible-playbook [-i /path/to/inventory] \
-    /usr/share/ansible/openshift-ansible/playbooks/deploy_cluster.yml
+    /usr/share/ansible/openshift-ansible/playbooks/deploy_router.yml
```

Comment:
The correct path is "/usr/share/ansible/openshift-ansible/playbooks/openshift-hosted/deploy_router.yml"